include/hw/i386/virt.h

```diff
@@ -67,6 +70,13 @@ typedef struct {
 
 MemoryRegion *virt_memory_init(VirtMachineState *vms);
 
-DeviceState *virt_acpi_init(qemu_irq *gsi, PCIBus *pci_bus);
+DeviceState *virt_acpi_init(qemu_irq *gsi, PCIBus **pci_bus, uint16_t segment_total);
+
+#define PCI_HOST_PCIEXBAR_BASE 0x40000000
```

Comment:
@liujing2 I like that these defines have now moved to the virt platform itself vs being PCI generic.

Reply:
@mcastelino  Yes, hotplug issue is the next thing I plan to do. I'll move that then.